web/vue.config.js

```diff
@@ -2,10 +2,14 @@ const MomentLocalesPlugin = require('moment-locales-webpack-plugin');
 const child_process = require('child_process');
 
 function getVersion() {
+  // Try to get the version via `git` if available; otherwise fall back on
+  // the COMMIT_REF environment variable provided by Netlify's build
+  // environment; if that is missing, report "unknown" as the version.
+
   try {
-    return child_process.execSync('git describe').toString();
+    return child_process.execSync('git describe --tags').toString();
   } catch (err) {
-    return process.env.COMMIT_REF;
+    return process.env.COMMIT_REF ? process.env.COMMIT_REF : 'unknown';
```

Comment:
Normally, you could use the nullish coalescing operator here, but I'm not sure whether NodeJS > 14 will be executing this in all build contexts, so perhaps this is safer.

Reply:
[Cool](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing_operator), didn't know about this. Thanks!